### ISSUE TITLE
Facebook error: An access token is required to request this resource.

### ISSUE THREAD

**Issue by cybercampbell** [iatek/jquery-socialist#68]
Hi

This was all previously working but today I noticed that it's no longer loading facebook images. 

When runnning on my site I get this error (firebug console):

http://graph.facebook.com/myfacebookpage/photos?limit=7 400 Bad Request 222ms

{
   "error": {
      "message": "An access token is required to request this resource.",
      "type": "OAuthException",
      "code": 104
   }
}

But if I run http://graph.facebook.com/myfacebookpage/photos?limit=7 directly in the browser it works fine.

I even tried your facebook page incase it was a permissions issue but I experinced the same results.

Any ideas on why this is happening.

Regards
C

**Comment by cybercampbell** [comment]
Seems the downloaded version from this site id out of date when compared to the online demo.

**Comment by topcatp800** [comment]
Hi @cybercampbell 

I am having the same issue and I have downloaded the master files from both this site and from:
http://iatek.github.io/jquery-socialist/

Both seems to have problems with facebook, when I looked at the example from:
http://plugins.in1.com/socialist/demo#social then it works with Facebook but the version of jquery.socialist.js is only version 1.0 9where as the downloaded version is now 1.01.

I remembered I had issues with version 1.0 with older browsers so what did you do to fix the issue?

Regards

T

**Comment by ghoobber** [comment]
Having the same issue. Looks like facebook stream is no longer works

**Comment by iatek** [comment]
Yes, it looks like FB may have switched the public 'photos' feed private. Would some else like to propose an auth based solution?

**Comment by link82** [comment]
I had the same issue two days ago then yesterday it started "working" again.

Even if the facebook page i'm pointing to has tons of posts i just receive a single post (3 years old).

I noticed that it seems ho have some problems with page names that are not in lowercase i.e. 'Grumpy-Cat' and even Gruppoicat i'm working on but works correctly with lowercase names like 'cocacola'.

I can't understand why, does it depends on the facebook page setup?

**Comment by Uraia** [comment]
According to facebook if the photos are public, any valid access token will do.

Had this issue as well, fixed it by adding a access token parameter to url and giving it a value ie. ?access_token='youraccesstoken'

To generate the token all you need to do is get the app id and app secret then generate the token, that is if you already have a facebook app created on the facebook developers section.

Go here https://smashballoon.com/custom-facebook-feed/access-token/ and follow those instructions if you're lost.

hope that helps you out.
